src/app/admin/components/reports/reports.directive.spec.js

```diff
@@ -504,6 +504,24 @@
                         expect(vm.searchedCertifiedProducts[0]).toEqual(expectedActivity);
                     });
 
+                    it('should handle criteria addition', function () {
```

Comment:
Did you run these tests? They shouldn't pass, as the html that's being output has the "bg-danger" class, which should not happen any more, as we're not marking things as questionable in the UI